workflow/comp_pred_ridge_and_LSTM.ipynb

```diff
@@ -34,23 +34,13 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        f"The MSE of climatology is {mean_squared_error(ground_truth, np.repeat(target_clim, ground_truth.anchor_year.size)):.3f}"
</pre><p>Thanks for correcting it. I forgot to update this with climatology. Btw, for a fair evaluation, I think when calculating the climatology, we shall only include the training data, instead of the whole dataset (if you are looking for something else, just ignore my comment).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/cookbook/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>